advisor-ui/src/components/Explore/Explore.css

```diff
@@ -0,0 +1,12 @@
+.explore {
+    align-items: flex-start;
+    display: flex;
+    flex-direction: column;
+}
+
+.explore > .content {
+    display: grid;
+    gap: 20px;
+    grid-template-columns: 200px 200px 200px 200px;
```

Comment:
I believe you can abbreviate this to `grid-template-columns: 200px;`, not actually 100% sure if it works for grid but I'd be surprised if not.